recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge,rapidsai
```

Comment:
Important: I think we should remove/revert this before merging?

Reply:
No. It doesn't matter because only optional deps/testing deps are pulled from the rapidsai channel.

Reply:
Can we use package from pypi?

Reply:
We can't use PYPI packages to satisfy the dependencies of a conda package because the package names are not guaranteed to be the same and second-order dependencies might break the environment.